src/main/java/uk/gov/companieshouse/model/Constants.java

```diff
@@ -12,7 +12,7 @@ private Constants() {
    public static final String PAYMENT_KIND = "dissolution-request#payment";
    public static final String PAYMENT_ITEM_KIND = "dissolution-request#payment-details";
    public static final String PAYMENT_RESOURCE_KIND = "dissolution-request#dissolution-request";
-   public static final String PAYMENT_DESCRIPTION = "Dissolution application";
+   public static final String PAYMENT_DESCRIPTION = "Apply to strike off and dissolve a company: %s (%s)";
```

Comment:
is there not already an approved branch for this in the JIRA comments?

Reply:
Yeah, but the branch is way behind master and wanted to create a new branch name for these changes so it matches with the user story number.